scss/mixins/_mixins.scss

```diff
@@ -5,6 +5,12 @@
     line-height: $fd-line-height;
     @include fd-var-color("color", $fd-color, --fd-color-text-1);
     font-family: $fd-font-family;
+    -webkit-font-smoothing: antialiased;
+    box-sizing: border-box;
+    &::before,
+    &::after {
+        box-sizing: inherit;
+    }
```

Comment:
These were previously being included on the `<html>` or `<body>` elements, which we can no longer depend on.